### DISCUSSION TITLE
Groups for GitHub Organizations

### DISCUSSION THREAD

**Discussion by kderck** [community/community#15250] [Other Feature Feedback, Questions, & Ideas]
Dear GitHub, 

One of my biggest frustrations with GitHub Organizations is that I am unable to create groups for repositories, I am able to assign groups for people but not for repositories. While, I understand that I can create another organisations for repositories it means that I need approval from the business to acquire a card.

Will GitHub look at creating groups for repositories, as once big monolith list even with search drives me crazy.

**Comment by AmirHmZz** [comment]
This is not an answer but I do need the same feature to categorize my organization repos. Currently I'm just able to do this with prefixing my repository names.
